.dockerignore

```diff
@@ -25,3 +25,5 @@ website/docs-pre-processed/
 !website/data
 
 tmp/
+
+.build/
```

Comment:
Hmm do we need to add this? I believe promu crossbuilds land in this dir

Reply:
Thx for catching this.